pyanaconda/modules/storage/bootloader/systemd.py

```diff
@@ -151,3 +151,11 @@ def install(self, args=None):
 
     def write_config_images(self, config):
         return True
+
+    def is_valid_stage1_device(self, device, early=False):
+        valid = True
+        if conf.system.provides_liveuser:
```

Comment:
Hi, we should probably make the special casing include also ostree based installations. However, I fear the logic needs to change a bit more thanks to that.

Reply:
Maybe it's more correct then to check the payload type? If the payload type is DNF then all good, otherwise error out.